server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -540,7 +540,7 @@ export default class ServiceProviderReferralsController {
 
     await this.scheduleAppointment(req, res, {
       getReferral: async () => referral,
-      getCurrentAppointment: async () => new SupplierAssessmentDecorator(supplierAssessment).currentAppointment,
+      getCurrentAppointment: async () => currentAppointment,
```

Comment:
where is `currentAppointment` defined? am I missing something?

Reply:
It's defined on line 539

Reply:
Ah, I was looking at the PR commit by commit, so it - it looks like this change was added in 7a89ff384 but the assignment on like 539 is added in the next commit in 093761ef40, hence the confusion. Ideally we'd have those two changes in at the same time, but don't worry too much about it now!

Reply:
ah I'm sorry about that.

I think the reason why this happened was because I did quite a bit of rebasing and I must have added it into the wrong commit.